android/CMakeLists.txt

```diff
@@ -13,18 +13,29 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON CACHE INTERNAL "")
 
 set (PACKAGE_NAME "reanimated")
+set (BUILD_DIR ${CMAKE_SOURCE_DIR}/build)
 set (SRC_DIR ${CMAKE_SOURCE_DIR}/src)
 
 if(${CLIENT_SIDE_BUILD})
     set (NODE_MODULES_DIR "${CMAKE_SOURCE_DIR}/../../")
     set (COMMON_SRC_DIR "${CMAKE_SOURCE_DIR}/../Common")
 else()
-    set (NODE_MODULES_DIR "../node_modules")
+    set (NODE_MODULES_DIR "../FabricExample/node_modules")
```

Comment:
Reanimated needs to link to the same shared libraries which are built from sources and located in `FabricExample/node_modules` instead of using pre-built shared libraries from `node_modules`.

This branch is only used for `FabricExample` app in our repository so it should not affect library users at all. However, this will break once we rename `FabricExample` to something else (which probably we won't). Anyway, we should generalize it by detecting `node_modules` path in `build.gradle` and passing it as parameter to `CMakeLists.txt`.

Reply:
> This branch is only used for `FabricExample` app in our repository so it should not affect library users at all. 

By this you mean that there will be another PR changing this file for the beta release?